server/sonar-server/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -362,7 +363,7 @@ private void completeFacets(Facets facets, SearchWsRequest request, Request wsRe
     if (actionPlansFromRequest != null) {
       actionPlans.addAll(actionPlansFromRequest);
     }
-    addMandatoryValuesToFacet(facets, ACTION_PLANS, actionPlans);
+    addMandatoryValuesToFacet(facets, DREPRECATED_ACTION_PLANS, actionPlans);
```

Comment:
A comment should be added in the "facets" field to explain that 'actionPlans' value is deprecated since 5.5